src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -50,6 +52,8 @@ public static string LocalSlotName
             }
         }
 
+#pragma warning disable SA1117 // Parameters should be on same line or separate lines
```

Comment:
Why do you need too disable this?

Reply:
It's for the `this.logger.LogInformation(...)` calls. I want to keep the long list of parameters on a single line underneath the template string so that we can easily compare the two and ensure we didn't add a parameter to one and not the other.